metadata.rb

```diff
@@ -4,10 +4,10 @@
 license 'Apache 2.0'
 description 'Installs/Configures packer'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version '1.4.3'
+version '1.4.4'
 
-depends 'ark', '~> 1.0.0'
-depends 'golang', '~> 1.7.0'
+depends 'ark'
+depends 'golang'
```

Comment:
I guess it would be safer to change `~>` to `>`, instead of removing it all together... At least it then remains backwards compatible.